src/main/java/racing/domain/Competition.java

```diff
@@ -0,0 +1,40 @@
+package racing.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Competition {
+    private final List<Car> cars;
+    private final int round;
+
+    public Competition(int unit, int round) {
+        cars = new ArrayList<>(unit);
+        for (int i = 0; i < unit; i++) {
+            cars.add(i, new Car());
+        }
+
+        this.round = round;
+    }
+
+    public void progressCompetition() {
+        for (int i = 0; i < round; i++) {
+            progressEachRound();
+        }
+    }
+
+    private void progressEachRound() {
+        for (int i = 0; i < cars.size(); i++) {
+            cars.get(i).accumulateOneLap();
+        }
```

Comment:
`enhanced for`를 사용하면 변수 `i`를 사용하지 않아도 될 것 같습니다.
```suggestion
        for (Car car : cars) {
            car.accumulateOneLap();
        }
```